src/quacc/atoms/skzcam.py

```diff
@@ -23,13 +26,751 @@
+        else 46
+        if x <= 54
+        else 54
+        if x <= 70
+        else 68
+        if x <= 80
+        else 78
+        if x <= 86
+        else 86
+        if x <= 102
+        else 100
+        if x <= 103
+        else None  # You can choose a default value or None for x > 103
+    )
+    for x in range(1, 104)  # Adjust the range to include up to 103
+    if x
+    in element_from_atomic_num_dict  # Ensure the atomic number exists in the dictionary
+}
+
+skzcam_cation_cap_ecp = {
+    "Ti": """NewECP
+N_core 0
+  lmax f
+  s 2
+   1      0.860000       9.191690  2
+   2      0.172000       0.008301  2
+  p 2
+   1      0.860000      17.997720  2
+   2      0.172000      -0.032600  2
+  d 2
+   1      1.600000      -9.504310  2
+   2      0.320000      -0.151370  2
+  f 1
+   1      1.000000000    0.000000000 2
+end""",
+    "Mg": """NewECP
+N_core 0
+lmax f
+s 1
+1      1.732000000   14.676000000 2
+p 1
+1      1.115000000    5.175700000 2
+d 1
+1      1.203000000   -1.816000000 2
+f 1
+1      1.000000000    0.000000000 2
+end""",
+}
+
 has_chemshell = find_spec("chemsh") is not None
 
 
+class SKZCAMInputSet:
+    """
+    A class to generate the complete set of SKZCAM inputs for performing direct MRCC and/or ORCA calculations. We define the possibility to have 4 ONIOM layers, each of which has a low-level (ll) and high-level (hl) calculation (see: doi.org/10.1021/jp962071j). It is assumed that the low-level for the subsequent ONIOM level is the high-level for the previous ONIOM level. The deltaCC calculation is also included as an option.
+
+    Attributes
+    ----------
+    adsorbate_slab_embedded_cluster
+        The ASE Atoms object containing the atomic coordinates and atomic charges from the .pun file, as well as the atom type. This object is created by the [quacc.atoms.skzcam.CreateSKZCAMClusters][] class.
+    quantum_cluster_indices_set
+        A list of lists containing the indices of the atoms of a set of quantum clusters. These indices are provided by the [quacc.atoms.skzcam.CreateSKZCAMClusters][] class.
+    ecp_region_indices_set
+        A list of lists containing the indices of the atoms in the ECP region of a set of quantum clusters. These indices are provided by the [quacc.atoms.skzcam.CreateSKZCAMClusters][] class.
+    mp2_oniom1_ll
+        A dictionary containing the information for the "low-level" MP2 treatment of ONIOM layer 1.
+    mp2_oniom1_hl
+        A dictionary containing the information for the "high-level" MP2 treatment of ONIOM layer 1.
+    mp2_oniom2_hl
+        A dictionary containing the information for the "high-level" MP2 treatment of ONIOM layer 2.
+    mp2_oniom3_hl
+        A dictionary containing the information for the "high-level" MP2 treatment of ONIOM layer 3.
+    mp2_oniom4_hl
+        A dictionary containing the information for the "high-level" MP2 treatment of ONIOM layer 4.
+    deltaCC
+        A dictionary containing the information for the deltaCC calculation.
```

Comment:
We should have just one of attributes or parameters to prevent duplication. I'd suggest just keeping the parameter docstrings.